ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -781,14 +793,16 @@ static void HandleKernelRequest(void* messageMemory, uint32_t messageSize)
             }
             cout << endl;
 #endif
+            bool isDirectory = requestHeader->messageType == MessageType_KtoU_NotifyDirectoryRenamed;
```

Comment:
There is no need to check `requestHeader->messageType == MessageType_KtoU_NotifyDirectoryRenamed` because we're in the `MessageType_KtoU_NotifyFileHardLinkCreated` case.